pom.xml

```diff
@@ -61,28 +61,28 @@
     <properties>
         <!-- Dependency Versions -->
         <bouncycastle.version>1.46</bouncycastle.version>
-        <greenmail.version>1.5.7</greenmail.version>
-        <hamcrest-core.version>1.3</hamcrest-core.version>
+        <greenmail.version>1.5.10</greenmail.version>
+        <hamcrest-core.version>2.2-rc1</hamcrest-core.version>
         <jacoco.version>0.8.1</jacoco.version>
-        <java-mail.version>1.6.1</java-mail.version>
-        <junit.jupiter.version>5.2.0</junit.jupiter.version>
-        <junit.platform.version>1.2.0</junit.platform.version>
-        <logback.version>1.2.3</logback.version>
-        <lombok.version>1.16.22</lombok.version>
-        <mockito.version>2.19.0</mockito.version>
-        <slf4j.version>1.7.25</slf4j.version>
+        <java-mail.version>1.6.2</java-mail.version>
+        <junit.jupiter.version>5.5.2</junit.jupiter.version>
+        <junit.platform.version>1.3.2</junit.platform.version>
+        <logback.version>1.3.0-alpha4</logback.version>
+        <lombok.version>1.18.10</lombok.version>
+        <mockito.version>3.0.0</mockito.version>
```

Comment:
Were you able to run all the tests successfully?

Reply:
Yes. UTs and ITs